### PR TITLE
Update office-add-in-availability.md

### PR DESCRIPTION
**Pull request by mattgeim**

### CODE REVIEW

File: docs/overview/office-add-in-availability.md [hunk 1]
```diff
@@ -17,6 +17,9 @@ To work as expected, your Office Add-in might depend on a specific Office applic
 > [!NOTE]
 > The initial Office 2016 release installed via MSI only contains the ExcelApi 1.1, WordApi 1.1, and Common API requirement sets. For more information about the update history of the various Office versions, check out the [See also](#see-also) section.
 
+> [!NOTE]
+> Office Add-ins may not be supported on all services that are members of the [Office Cloud Storage Partner Program](https://developer.microsoft.com/en-us/office/cloud-storage-partner-program), which enables integrating Office on the web to work with Office documents as part of their service offering. For more information, please contact the member service.
```
Comment (lindalu-MSFT) [inline docs/overview/office-add-in-availability.md:21]:
```suggestion
> Office Add-ins may not be supported on all services that are members of the [Office Cloud Storage Partner Program](https://developer.microsoft.com/office/cloud-storage-partner-program), which enables integrating Office on the web to work with Office documents as part of their service offering. For more information, please contact the member service.
```

File: docs/overview/office-add-in-availability.md [hunk 2]
```diff
@@ -17,6 +17,9 @@ To work as expected, your Office Add-in might depend on a specific Office applic
 > [!NOTE]
```
Comment (ElizabethSamuel-MSFT) [inline docs/overview/office-add-in-availability.md:17]: Update ms.date metadata at top of file

File: docs/overview/office-add-in-availability.md [hunk 3]
```diff
@@ -17,6 +17,9 @@ To work as expected, your Office Add-in might depend on a specific Office applic
...
 > The initial Office 2016 release installed via MSI only contains the ExcelApi 1.1, WordApi 1.1, and Common API requirement sets. For more information about the update history of the various Office versions, check out the [See also](#see-also) section.
 
+> [!NOTE]
```
Comment (ElizabethSamuel-MSFT) [inline docs/overview/office-add-in-availability.md:20]: Should we have two consecutive notes? Maybe consider merging into one note (2 parts, of course) or changing from NOTE to IMPORTANT.

Comment (lindalu-MSFT) [reply docs/overview/office-add-in-availability.md:20]: I'd combine to one note with 2 paragraphs since both have to do with versions.

**Comment by AlexJerabek** [comment]
@davidchesnut,
Since @mattgeim is out, can you drive this PR to completion?

**Pull request merged**